website/docs/api/themes/theme-configuration.md

```diff
@@ -216,26 +235,13 @@ module.exports = {
       // highlight-start
       items: [
         {
-          // Client-side routing, used for navigating within the website.
-          // The baseUrl will be automatically prepended to this value.
           to: 'docs/introduction',
-          // A full-page navigation, used for navigating outside of the website.
-          // You should only use either `to` or `href`.
-          href: 'https://www.facebook.com',
-          // Prepends the baseUrl to href values.
-          prependBaseUrlToHref: true,
-          // The string to be shown.
+          // Only one of "to" or "href" should be used
```

Comment:
seems fine to remove as it's already explained in the table

Reply:
I think this improves readability than having a lone commented-out `href`? Readers should be able to comprehend every line on one glance.